manifests/guestbook.yaml

```diff
@@ -1,34 +1,54 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: guestbook
+  name: go-guestbook
```

Comment:
Remove the `go-` prefix everywhere

Reply:
I added those to fix? an issue where deployment is not found during okteto up